packages/wrapper/src/editorApp.ts

```diff
@@ -37,6 +38,16 @@ export interface CodeResources {
     original?: CodeContent;
 }
 
+export interface CallbackDisposeable {
```

Comment:
Disposable :)

Reply:
Will fix that

Reply:
Force pushed. I found some other left-overs ⬇️